saito-js/lib/block.ts

```diff
@@ -29,31 +27,32 @@ export default class Block extends WasmWrapper<WasmBlock> {
         hash: this.hash,
         type: JSON.stringify(this.block_type),
         previous_block_hash: this.previousBlockHash,
-        transactions: this.transactions.map((tx) => tx.toJson()),
-        cv: this.instance.cv,
+        txs: this.instance.transactions,
         it_index: this.instance.it_index,
```

Comment:
why was cv removed?

Reply:
I added the properties of cv to the block object instead of putting it inside cv